oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/util.js

```diff
@@ -56,3 +60,53 @@ export const eliminateUsedLanguages = (
   );
   return remainingLanguages;
 };
+
+// function that goes through an object and all its members and removes all the keys in the
+// list of keys provided as argument from any object that lives inside the initial object,
+//  even if it is inside of an array
+
+function removeKeyFromNestedObjects(obj, keysToRemove) {
```

Comment:
Is this func used from anywhere? (e.g. record serializer)

Reply:
@mirekys yes it is used in the serializer, I forgote to delete it from utils, my bad.